docs/source/metrics.rst

```diff
@@ -1,4 +1,334 @@
-.. automodule:: pytorch_lightning.metrics
-   :members:
-   :noindex:
-   :exclude-members:
+Metrics
+=======
+
+Metrics are used to monitor model performance.
+
+In this package we provide two major pieces of functionality.
+
+    1. A Metric class you can use to implement metrics with built-in distributed (ddp) support which are device agnostic.
+    2. A collection of popular metrics already implemented for you.
+
```

Comment:
Should we mention that the package also comes with a interface to sklearn metrics (with a warning that this is slow due to casting back-and-forth of the tensors)

Reply:
We should. We should also include that sklearn has to be installed separately

Reply:
We should make it clear how to import the different backends. Maybe something like:

Use native backend
`import pytorch_lightning.metrics.native as plm`

Use sklearn backend
`import pytorch_lightning.metrics.sklearn as plm`

Use default (native if available else sklearn)
`import pytorch_lightning.metrics as plm`

Reply:
@SkafteNicki can you add the sklearn details in here?